crowbar_framework/config/locales/suse_manager_client/en.yml

```diff
@@ -22,3 +22,12 @@ en:
         manager_server_hint: 'For example susemgr.example.com. You can set a record within the %{link}.'
         activation_key: 'Activation Key'
         activation_key_hint: 'For example 1-00112233445566778899aabbccddeeff'
+
+        manual_steps: |
+          In order to use this barclamp you have to prepare some manual steps.
+          After installing the SUSE Manager Server you have to download the RPM
+          from <strong>%{manager_url}</strong> (check the version number) and
```

Comment:
I know it was like that before, but I have no clue about why `check the version number`. Maybe we should just drop that bit if we can't explain it?